.github/workflows/reusable_basic.yml

```diff
@@ -124,7 +124,7 @@ jobs:
     - name: Install apt packages
       run: |
         sudo apt-get update
-        sudo apt-get install -y clang cmake libnuma-dev lcov
+        sudo apt-get install -y clang cmake libhwloc-dev libnuma-dev lcov
```

Comment:
I think HWLOC should be installed only when disable_hwloc == OFF ?

Reply:
It should, but it wasn't. It was always installed, so I've left the logic intact in this PR. Anyway, the proposed change is a small one, I'll include it in this PR.

Reply:
Done